src/Compute/Compute/Generated/Snapshot/Config/NewAzureRmSnapshotConfigCommand.cs

```diff
@@ -186,6 +186,12 @@ public partial class NewAzureRmSnapshotConfigCommand : Microsoft.Azure.Commands.
         [PSArgumentCompleter("X64", "Arm64")]
         public string Architecture { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Required if createOption is CopyFromSanSnapshot. this is the ARM id of the source elastic san volume snapshot.")]
```

Comment:
Capital This

Reply:
Fix with https://github.com/Azure/azure-powershell/pull/23136.